main-site/components/faq-section/FaqSection.styles.ts

```diff
@@ -0,0 +1,78 @@
+import styled from 'styled-components';
+import { max } from '../../../shared-ui/lib/responsive';
+import { colors } from '../../../shared-ui/style/colors';
+import { H4, P } from '../../../shared-ui/style/typography';
+
+const StyledFaqSectionContainer = styled.div`
+  text-align: center;
+  position: relative;
+  top: 2em;
+  margin-bottom: 5em;
+`;
+const StyledCactus = styled.img`
+  position: absolute;
+  margin-left: 5em;
+  top: 0.1em;
+  left: 1em;
+
+  @media ${max.tablet} {
+    width: 13em;
+    top: -1em;
+    display: none;
+  }
+`;
+const StyledH4 = styled(H4)`
+  color: ${colors.TEXT_BROWN};
+  margin-top: 3em;
+  padding: 2em;
+`;
+const StyledFaqContainer = styled.div`
+position: relative;
+width: 60%;
+margin: 0 auto;
+color: ${colors.TEXT_BROWN}
+position: relative;
+padding: 4em;
```

Comment:
I think you need to remove this padding for mobile / smaller screen sizes since it's pushing the container out!
https://user-images.githubusercontent.com/59738880/194614872-795e38aa-20a8-429e-99c7-a0a31e3bca77.mov